### ISSUE TITLE
React.renderToStaticMarkup -> ReactDOMServer.renderToStaticMarkup

### ISSUE THREAD

**Issue by leebenson** [ericclemmons/react-resolver#101]
Getting this warning in the console:

```
Warning: React.renderToStaticMarkup is deprecated. Please use ReactDOMServer.renderToStaticMarkup from require('react-dom/server') instead.
```

I saw the todo @ [react-resolver/blob/master/src/Resolver.js#L49](https://github.com/ericclemmons/react-resolver/blob/master/src/Resolver.js#L49), but just a friendly reminder :+1:

**Comment by ericclemmons** [comment]
This will be fixed by #100.

I need to just get on it & merge it down :(

**Comment by pwmckenna** [comment]
#100 was merged and released as part of [v3.0.0](https://www.npmjs.com/package/react-resolver). Shouldn't see those deprecation warnings anymore! :evergreen_tree:
